src/common/containers/build-details.js

```diff
@@ -22,6 +22,27 @@ class BuildDetails extends Component {
     const { repository, buildId, build } = this.props;
     const { snap, error, isFetching } = this.props.snapBuilds;
 
+    const buildFailed = (build.statusMessage === 'Failed to build');
```

Comment:
Currently the only 'status' we have for build is a message from LP, so we need to do a string comparison to detect particular status.